### PR TITLE
Add Enable Auth Select to RHEL8/9 STIG

### LINKED ISSUE #9142: STIG profile can't be properly remediated on machine installed with `tests/kickstarts/test_suite.cfg`
#### Description of problem:
`tests/kickstarts/test_suite.cfg` kickstart doesn't `authselect select sssd` anymore, because `enable_authselect` rule has been introduced to content and added to almost all profiles except STIG.

#### SCAP Security Guide Version:
11974e4
#### Operating System Version:
RHEL8, RHEL9

#### Steps to Reproduce:
1. Install machine with `tests/kickstarts/test_suite.cfg`
2. For example, run STIG ansible playbook

#### Actual Results:
```
TASK [Ensure PAM Displays Last Logon/Access Notification - Check integrity of authselect current profile] ***
fatal: [192.168.122.38]: FAILED! => {"changed": false, "cmd": ["authselect", "check"], "delta": "0:00:00.006103", "end": "2022-07-09 09:58:59.539972", "msg": "non-zero return code", "rc": 2, "start": "2022-07-09 09:58:59.533869", "stderr": "", "stderr_lines": [], "stdout": "System was not configured with authselect.", "stdout_lines": ["System was not configured with authselect."]}
...ignoring

TASK [Ensure PAM Displays Last Logon/Access Notification - Informative message based on the authselect integrity check result] ***
fatal: [192.168.122.38]: FAILED! => {
    "assertion": "result_authselect_check_cmd is success",
    "changed": false,
    "evaluated_to": false,
    "msg": [
        "authselect integrity check failed. Remediation aborted!",
        "This remediation could not be applied because an authselect profile was not selected or the selected profile is not intact.",
        "It is not recommended to manually edit the PAM files when authselect tool is available.",
        "In cases where the default authselect profile does not cover a specific demand, a custom authselect profile is recommended."
    ]
}
```
because of `authselect` profile has been selected

#### Expected Results:
STIG profile rules can be remediated when installed using `tests/kickstarts/test_suite.cfg`

#### Additional Information/Debugging Steps:
`enable_authselect` should be included to STIG and STIG with GUI profiles. However, the rule doesn't have STIG ID nor SRG.

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

* Add the rule enable_authselect to the RHEL8 and RHEL9 STIG profiles
* Add references for enable_authselect

#### Rationale:

Fixes #9142

cc @mildas

### CODE REVIEW

File: products/rhel9/profiles/stig.profile [hunk 1]
```diff
@@ -81,7 +81,9 @@ selections:
     - configure_bind_crypto_policy
     - configure_libreswan_crypto_policy
     - configure_kerberos_crypto_policy
-    - enable_dracut_fips_module
```
Comment (mildas) [inline products/rhel9/profiles/stig.profile:84]: Why it has been removed?

**Review by mildas — CHANGES_REQUESTED** [review]
Define `authselect` profile via variable - https://github.com/ComplianceAsCode/content/blob/master/products/rhel8/profiles/ospp.profile#L223

`authselect select sssd` is not needed in kickstart anymore. Please remove it too

**Review by marcusburghardt — APPROVED** [review]

**Comment by Mab879** [comment]
The error on the Rawhide failure is on the Fedora side, not caused by this change.

**Comment by mildas** [comment]
LGTM

**Pull request merged**